src/tests/suite/daos_test.c

```diff
@@ -231,6 +232,14 @@ run_specified_tests(const char *tests, int rank, int size,
 			nr_failed += run_daos_nvme_recov_test(rank, size,
 						sub_tests, sub_tests_size);
 			break;
+		case 'v':
+			daos_test_print(rank, "\n\n=================");
+			daos_test_print(rank, "DAOS rebuild simple tests..");
+			daos_test_print(rank, "=================");
+			nr_failed += run_daos_rebuild_simple_test(rank, size,
+					   	 sub_tests, sub_tests_size);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs